package.json

```
@@ -63,6 +63,7 @@
     "test:lsp11": "hardhat test --no-compile tests/LSP11BasicSocialRecovery/LSP11BasicSocialRecovery.test.ts",
     "test:lsp11init": "hardhat test --no-compile tests/LSP11BasicSocialRecovery/LSP11BasicSocialRecoveryInit.test.ts",
     "test:lsp17": "hardhat test --no-compile tests/LSP17ContractExtension/LSP17Extendable.test.ts",
+    "test:lsp17extensions": "hardhat test --no-compile tests/LSP17Extentions/**/*.test.ts",
```

Typo for the folder name:

```suggestion
    "test:lsp17extensions": "hardhat test --no-compile tests/LSP17Extensions/**/*.test.ts",
```